doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -186,12 +179,58 @@ The procedure to install CORTX on OVA is mentioned below.
    
     sh /opt/seagate/cortx/s3/scripts/s3-sanity-test.sh -e 127.0.0.1
 
-    * The script performs several operations on S3 API and LDAP backend:
-      create account
-      create user
-      create bucket
-      put object
-      delete all the above in reverse order
+      Check S3CMD...OK
+      Check s3iamcli...OKconfigured s3iamcli
+      using s3endpoint 127.0.0.1
+      /root/.sgs3iamcli/config.yaml
+      /root/.s3cfg
+
+      Sanity Account doesn't exist for cleanup
+      using s3endpoint 127.0.0.1
+      /root/.sgs3iamcli/config.yaml
+      /root/.s3cfg
+
+
+      *** S3 Sanity ***
+
+      **** Create Account *******
+      UserId = AIDAAA9B35A55DFF441AA, ARN = arn:aws:iam::548273190390:user/SanityUserToDeleteAfterUse, Path = /
```

Comment:
@mukul-seagate11  Not sure if we should paste S3 Sanity output in doc. It's printing UuserId and other data. Though I believe this is instance-specific data.

Reply:
OK, corrected

Reply:
@mukul-seagate11  - Can we keep this info. As it describes what the sanity test is doing instead of the command output. 

```
    * The script performs several operations on S3 API and LDAP backend:
      create account
      create user
      create bucket
      put object
      delete all the above in reverse order
```

Reply:
done